### PR TITLE
i18n(pt-BR): Adding `install/auto.md` page

### PR DESCRIPTION
**Pull request by caioferrarezi**
Adding the `install/auto.md` page in pt-BR and `nav.ts` text for it as well!

### CODE REVIEW

File: src/pages/pt-BR/install/auto.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+## Próximos Passos
+
+Succeso! Agora você está pronto para começar a desenvolver!
+
+📚 Aprenda mais sobre a estrutura de projeto Astro em nosso [Guia de Estrutura de Projeto](/en/core-concepts/project-structure).
+
+📚 Aprenda mais sobre a sintaxe de componente Astro em nosso [Guia de Componentes Astro](/en/core-concepts/astro-components).
+
+📚 Aprenda mais sobre rotas baseadas em arquivos Astro em nosso [Guia de Rotas](/en/core-concepts/astro-pages).
```
Comment (Yan-Thomas) [inline src/pages/pt-BR/install/auto.md:148]:
```suggestion
📚 Aprenda mais sobre a estrutura de projetos Astro em nosso [Guia de Estrutura de Projeto](/pt-BR/core-concepts/project-structure).

📚 Aprenda mais sobre a sintaxe de componentes Astro em nosso [Guia de Componentes Astro](/pt-BR/core-concepts/astro-components).

📚 Aprenda mais sobre rotas baseadas em arquivos Astro em nosso [Guia de Rotas](/pt-BR/core-concepts/astro-pages).
```

File: src/pages/pt-BR/install/auto.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+# yarn
+yarn build
+
+# pnpm
+pnpm run build
+```
+
+Quando o comando finalizar, você deverá ter uma nova pasta `dist/` em seu projeto que poderá ser hospedada em seu serviço de hospedagem favorito.
+
+Para iniciar a hospedagem gratuita do seu site, conheça o nosso parceiro de hospedagem [Netlify](https://www.netlify.com/). Para instruções de deploy em qualquer serviço de hospedagem que você escolher, leia o nosso detalhado [guia de deploy](/en/guides/deploy).
```
Comment (Yan-Thomas) [inline src/pages/pt-BR/install/auto.md:138]: Added the pt-BR link and some small changes:

```suggestion
Para iniciar a hospedagem gratuita do seu site, conheça o nosso parceiro de hospedagem [Netlify](https://www.netlify.com/). Para instruções de deploy no serviço de hospedagem que preferir, leia o nosso detalhado [guia de deploy](/pt-BR/guides/deploy).
```

File: src/pages/pt-BR/install/auto.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+> 💡 Quer conhecer os temas antes de escolher? Visite: [astro.new](https://astro.new/)
+
+### Instale as dependências (opcional)
+O assistente de instalação irá oference para executar o comando `install` neste momento para você, etapa opcional.
+
+> ⚠️ Se você não deseja fazê-lo neste momento, será necessário [instalar as dependências](/en/install/auto#2-install-dependencies) após o assistente finalizar e antes de iniciar o seu projeto.
+
+### Instale qualquer Integração Oficial Astro (opcional)
+Você terá nesse momento a opção de adicionar qualquer [framework de UI adicional](/en/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact, Lit) e também qualquer Integração Oficial Astro (Tailwind, Turbolinks, Partytown, Sitemap) ao executar `astro add --yes`.
```
Comment (Yan-Thomas) [inline src/pages/pt-BR/install/auto.md:58]: Add pt-BR link:

```suggestion
Você terá nesse momento a opção de adicionar qualquer [framework de UI adicional](/pt-BR/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact, Lit) e também qualquer Integração Oficial Astro (Tailwind, Turbolinks, Partytown, Sitemap) ao executar `astro add --yes`.
```

**Review by Yan-Thomas — COMMENTED** [review]
Thanks for the translations! I added some comments/opinions/suggestions here and there.

File: src/pages/pt-BR/install/auto.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+<InstallGuideTabGroup />
+
+#### Instalação
+
+`create-astro` é a forma mais fácil e rápida de configurar um novo projeto Astro do zero.
+
+## 1. Execute a Interface de Linha de Comando
+
+Execute o seguinte comando em seu terminal para iniciar o nosso assistente de instalação *wizard*, `create-astro`. Isso irá guiá-lo na criação do seu primeiro projeto Astro.
```
[the remark this answers is not in the document]
Comment (caioferrarezi) [reply src/pages/pt-BR/install/auto.md:25]: That's perfect! I wasn't sure about this sentece, but this solves the problem!

File: src/pages/pt-BR/install/auto.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+- `Completely empty`: Um template contendo apenas o mínimo para iniciar.
+
+Use as teclas direcionais (cima e baixo) para navegar para o template que deseja instalar, então pressione *enter* para selecionar.
+
+> 💡 Quer conhecer os temas antes de escolher? Visite: [astro.new](https://astro.new/)
+
+### Instale as dependências (opcional)
+O assistente de instalação irá oference para executar o comando `install` neste momento para você, etapa opcional.
+
+> ⚠️ Se você não deseja fazê-lo neste momento, será necessário [instalar as dependências](/en/install/auto#2-install-dependencies) após o assistente finalizar e antes de iniciar o seu projeto.
```
[the remark this answers is not in the document]
Comment (caioferrarezi) [reply src/pages/pt-BR/install/auto.md:55]: Oh, I didn't know that! Thanks!

File: src/pages/pt-BR/install/auto.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+- `Just the basics`: Um ótimo template inicial para quem busca explorar Astro.
+- `Blog`, `Documentation`, `Portfolio`: Temas pré-configurados para casos de uso específicos.
+- `Completely empty`: Um template contendo apenas o mínimo para iniciar.
+
+Use as teclas direcionais (cima e baixo) para navegar para o template que deseja instalar, então pressione *enter* para selecionar.
+
+> 💡 Quer conhecer os temas antes de escolher? Visite: [astro.new](https://astro.new/)
+
+### Instale as dependências (opcional)
+O assistente de instalação irá oference para executar o comando `install` neste momento para você, etapa opcional.
```
[the remark this answers is not in the document]
Comment (caioferrarezi) [reply src/pages/pt-BR/install/auto.md:53]: I'm sorry about all typos 😅, I did it quickly and didn't have so much to review it myself. Thank once again!

File: src/pages/pt-BR/install/auto.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+yarn
+
+# pnpm
+pnpm install
+
+```
+
+## 3. Inicie Astro ✨
+
+Você pode utilizar o servidor de desenvolvimento integrado ao Astro para a construção do seu projeto. Esta é maneira de executar o seu projeto durante o desenvolvimento local.
```
[the remark this answers is not in the document]
Comment (caioferrarezi) [reply src/pages/pt-BR/install/auto.md:100]: No problems at all, I agree!

File: src/pages/pt-BR/install/auto.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+# yarn
+yarn start
+
+# pnpm
+pnpm start
+```
+
+Se tudo ocorrer bem, Astro deverá servir o seu projeto em [http://localhost:3000](http://localhost:3000)!
+
+Astro irá observar as mudanças que ocorrerem em seu diretório `src/` e então irá reiniciar o servidor local assim que houver mudanças durante o desenvolvimento.
```
[the remark this answers is not in the document]
Comment (caioferrarezi) [reply src/pages/pt-BR/install/auto.md:117]: Maybe `conforme as mudanças aconteçam durante o desenvolvimento`?

**Comment by caioferrarezi** [comment]
Thanks for reviewing @Yan-Thomas! This was my first page ever 😄 

I just left one last comment to your suggestion

File: src/pages/pt-BR/install/auto.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (Yan-Thomas) [reply src/pages/pt-BR/install/auto.md:117]: I agree!

**Comment by delucis** [comment]
@Yan-Thomas If this is ready to merge, could you post a “LGTM” (Looks good to me)? That way we’ll be sure it’s good to go.

Thanks both of you for your hard work putting this together! 💜

File: src/pages/pt-BR/install/auto.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
+---
+title: Instale Astro com a Interface de Linha de Comando Automática.
```
Comment (delucis) [inline src/pages/pt-BR/install/auto.md:2]: Oh, one detail actually! I guess there should not be a `.` at the end of the title right?

```suggestion
title: Instale Astro com a Interface de Linha de Comando Automática
```

**Comment by Yan-Thomas** [comment]
Sure. When it's ready I'm going to approve and comment "LGTM"!

**Comment by Yan-Thomas** [comment]
I think it's complete now! LGTM (let's get this merge) 🤣

**Review by delucis — APPROVED** [review]
💰💰💰💰

**Pull request merged**